paasta_tools/cli/utils.py

```diff
@@ -742,3 +744,52 @@ def pick_slave_from_status(status, host=None):
     else:
         slaves = status.marathon.slaves
         return slaves[0]
+
+
+class TaskNotFound(Exception):
```

Comment:
I'm renaming to `PaastaTaskNotFound` as technically the task is a different object to the one mesos-cli produces.